src/com/rcv/ResultsWriter.java

```diff
@@ -517,16 +519,16 @@ private int addHeaderRows(XSSFSheet worksheet, Map<Integer, BigDecimal> totalAct
       org.apache.poi.ss.usermodel.Row row = worksheet.createRow(rowCounter++);
       // create a cell if any text in the first element
       if (rowFields[0] != null) {
-        row.createCell(0).setCellValue((String)rowFields[0]);
+        row.createCell(0).setCellValue((String) rowFields[0]);
```

Comment:
Oops. You've clearly read the style guidelines more closely than we did!

Reply:
All credit goes to IntelliJ!